inference-engine/tests/functional/inference_engine/ir_serialization/ordering.cpp

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <fstream>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "gtest/gtest.h"
+#include "ie_core.hpp"
+#include "pugixml.hpp"
+
+#ifndef IR_SERIALIZATION_MODELS_PATH  // should be already defined by cmake
+#define IR_SERIALIZATION_MODELS_PATH ""
+#endif
+
+namespace {
+// walker traverse (DFS) xml document and store layer type attriubte in
+// vector which is later used for comparison
+struct type_attribute_walker : pugi::xml_tree_walker {
+    std::vector<pugi::xml_attribute> attributes;
+
+    virtual bool for_each(pugi::xml_node& node) {
```

Comment:
If this method is virtual in the xml_tree_walker class, it should be marked as override here. Could you check it?

Reply:
Applied in [this](https://github.com/openvinotoolkit/openvino/pull/3156/commits/1b80d2856153c7ccad62f8cd9146934a34529b23) commit.